OptionSelector/OptionsTableViewController.h

```diff
@@ -0,0 +1,16 @@
+//
+//  OptionsTableViewController.h
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CQCategory.h"
```

Comment:
Style: It's generally better to keep `#import` directives in the .m files. You can get around having it here by including the following in your .h file instead:
```objc
@class CQCategory;
```
Just make sure you include the `#import` in the .m file.